mteb/tasks/Retrieval/eng/ARCChallengeRetrieval.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class ARCChallenge(AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        name="ARCChallenge",
+        description="Reasoning as Retrieval (RAR-b) format: Whether Answers to Queries in Reasoning Tasks can be retrieved as top.",
+        reference="https://allenai.org/data/arc",
+        dataset={
+            "path": "RAR-b/ARC-Challenge",
+            "revision": "c481e0da3dcbbad8bce7721dea9085b74320a0a3",
+        },
+        type="Retrieval",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["eng-Latn"],
+        main_score="ndcg_at_10",
+        date=("2018-01-01", "2018-12-31"),
+        form=["written"],
+        domains=["Encyclopaedic"],
+        task_subtypes=[],
```

Comment:
Sounds like this would be reasonable (you will have to add it to the list of allowed subtypes)
```suggestion
        task_subtypes=["Reasoning as Retrieval"],
```